### DISCUSSION TITLE
Restore Windows Explorer command bar

### DISCUSSION THREAD

**Discussion by KineticTactic** [valinet/ExplorerPatcher#131] [Q&A] [answered]
I have renamed `dxgi.dll` in `C:\Windows` to `dxgio.dll`. Everything such as the taskbar and right click context menus have gone back to normal, however the windows explorer still remained the same (the new command bar does not appear).
![image](https://user-images.githubusercontent.com/63810712/137584520-78b344e7-651b-47fe-baea-8589a8472f48.png)
I have also noticed that snap layouts no longer work.

I have tried the **Restore Explorer command bar** command from [here](https://www.reddit.com/r/Windows11/comments/pu5aa3/howto_disable_new_context_menu_explorer_command/) but to no avail.

**Comment by KineticTactic** [comment, marked as answer]
Sorry, I had used ThisIsWin11 which had installed ExplorerPatcher, and since ExplorerPatcher had this option i had assumed that it was an issue with ExplorerPatcher.
However ThisIsWin11 had actually hidden the command bar, and restoring the option from there has fixed this issue

**Reply by valinet** [reply]
Yeah, also, the tweaks to disable the command bar and menu are permanent, do not revert on uninstall. Better to restore default settings, then uninstall.
